doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2049,3 +2049,12 @@ Allows Cross-Origin Resource sharing(CORS). By default this setting is absent an
 If you don’t want to allow CORS for your installation, set:
 
 ``curl -X PUT -d 'false' http://localhost:8080/api/admin/settings/:AllowCors``
+
+:ChronologicalDateFacets
++++++++++++++++
```

Comment:
Just submitted PR #6998 that cleaned up some format warnings when building the sphinx-guides. This header formatting is gonna trip `WARNING: Title underline too short` cause there aren't enough `+` characters to match the length of the `:ChronologicalDateFacets` title text.

Reply:
fixed - thanks!

Reply:
👍